# Array of detected objects
Header header
DetectedObject[] objects
